src/commands/init.js

```diff
@@ -30,9 +31,9 @@ const templates = [{
  *
  * @returns {Promise} - Promise for the command.
  */
-export default function init({ parsedArguments, parsedOptions }) {
+export default function init({ parsedArguments, parsedOptions, directory }) {
     const { list, force } = parsedOptions.options;
-    const { name, template, version } = parsedArguments.arguments;
+    const { template, version } = parsedArguments.arguments;
```

Comment:
Ah, so much clearer when you RTFM. ;) So that raises a question. Do we want name to override the directory option or should name be joined with the directory?
I vote for:
```javascript
return checkFolder(force, name || directory).then((dir) => {
```